### ISSUE TITLE
Mission Profile Issues

### ISSUE THREAD

**Issue by TakeshiYamato01** [bcholmes/StarTrek2d20#115]
So, a few things regarding Mission Profiles from Utopia Planitia aren't mirrored in the app yet.

First, the Utopia Planitia book does give Federation Starships access to the Warship Profile, just like Klingon ships. The stats are exactly the same, aside from what will be the second issue.

This second issue is that, in any Mission Profile which originally had the Quantum Torpedoes Talent (namely Tactical Operations and Warship), Utopia Planitia replaces that Talent with Expanded Munitions, which is more universally applicable.

And the third issue is this line from the intro to the Mission Profiles section: "Gamemasters and players are encouraged to amend these mission profiles or create their own based on their game's needs and desires."

So, according to that line, we should probably have an option for custom Mission Profiles, just like custom Spaceframes.

**Comment by TakeshiYamato01** [comment]
Still haven't seen any changes here. Any progress made on this yet? Been almost 7 months, so I figured I'd check in.

**Comment by bcholmes** [comment]
I think I've dealt with the first two of these issues. Custom Mission Profiles is a much bigger requirement.

**Comment by TakeshiYamato01** [comment]
Yep, the first two issues are handled. I agree about custom mission profiles, but the Warship and Tac Ops profiles have been appropriately adjusted, which is a good thing.
